lib/scarpe/html.rb

```diff
@@ -8,7 +8,7 @@ def self.render(&block)
 
     def initialize(&block)
       @buffer = ""
-      @buffer += block.call(self)
```

Comment:
nit: we should probably initialize the buffer to whatever we get from the call - I will do that change in a couple of minutes (I am going to write a couple of tests for the html module)